build/includes/terraform.mk

```diff
@@ -83,3 +84,11 @@ endif
 gcloud-terraform-destroy-cluster:
 	$(DOCKER_RUN) bash -c 'cd $(mount_path)/build/terraform/gke && \
 	 terraform destroy -target module.helm_agones.helm_release.agones -auto-approve && sleep 60 && terraform destroy -auto-approve'
+
+terraform-test: $(ensure-build-image)
```

Comment:
Curious, do we plan to run this as part of cloud build?

Reply:
Mark, there is no way to run this as we need a permission to create GKE cluster from tests in Cloud Build.
But if we limit this test to verify helm templates (not terraform configs) then it is possible.

Reply:
I think we could not create GKE cluster from inside Cloud Build this is not safe and could lead to cluster leaking.